templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>You want delete {{ car }}</h1>
```

Comment:
The message 'You want delete {{ car }}' is missing a word. It should be 'You want to delete {{ car }}' for correct grammar.